builtin/mainmenu/content/pkgmgr.lua

```diff
@@ -711,6 +711,53 @@ function pkgmgr.preparemodlist(data)
 	return retval
 end
 
+function pkgmgr.prepareclientmodlist(data)
+	local retval = {}
+
+	local clientmods = {}
+
+	--read clientmods
+	local modpath = core.get_clientmodpath()
+
+	if modpath ~= nil and modpath ~= "" then
+		pkgmgr.get_mods(modpath, "clientmods", clientmods)
+	end
+
+	for i=1,#clientmods,1 do
+		clientmods[i].type = "mod"
+		clientmods[i].loc = "global"
+		clientmods[i].is_clientside = true
+		retval[#retval + 1] = clientmods[i]
+	end
+
+	--read mods configuration
+	local filename = modpath ..
+				DIR_DELIM .. "mods.conf"
+
+	local conffile = Settings(filename)
+
+	for key,value in pairs(conffile:to_table()) do
+		if key:sub(1, 9) == "load_mod_" then
+			key = key:sub(10)
+			local element = nil
+			for i=1,#retval,1 do
+				if retval[i].name == key and
+					not retval[i].is_modpack then
+					element = retval[i]
+					break
+				end
+			end
+			if element ~= nil then
+				element.enabled = value ~= "false" and value ~= "nil" and value
```

Comment:
```suggestion
				element.enabled = value ~= "false" and value ~= "nil"
```

i don't see `and value` adding any value, it just creates type confusion: the `value` is a string and thus necessarily truthy.